peerprep/app/questions/[question]/question.tsx

```diff
@@ -77,34 +71,16 @@ function QuestionBlock({ question }: Props) {
           </PeerprepButton>
         </div>
         <div className={styles.label_wrapper}>
-          <p>Categories: </p>
-          {question.categories.length == 0 ? (
-            <p>No categories listed.</p>
+          <p>Topics: </p>
+          {question.topicTags.length == 0 ? (
+            <p>No topics listed.</p>
           ) : (
-            question.categories.map((elem, idx) => <p key={idx}>{elem}</p>)
+            question.topicTags.map((elem, idx) => <p key={idx}>{elem}</p>)
           )}
         </div>
-        <p>{question.description}</p>
+
+        { <div dangerouslySetInnerHTML={{ __html: DOMPurify.sanitize(question.content)}} /> }
```

Comment:
Why do we need to DOMPurify this? I don't quite follow why we are storing the HTML tags in content directly.

Reply:
Carryover from leetcode api

Reply:
Then we probably need a way to inject the HTML input for making a new question for formatting - fairly certain it should be fine still without modifications though. I'll raise an issue with a nice-to-have for now.